Documentation/WritingReST/Reference/Code/Confval.rst

```diff
@@ -28,49 +27,175 @@ Each configuration value name may only be used once. In large references
 with different contexts you can define individual configuration schemas for
 each context.
 
+..  contents:: Table of contents
```

Comment:
I would prefer moving examples after each confval directive or behind all directives, but not in front. You have inserted a very detailed example for fast copy&paste in CheatSheet.

Reply:
It seems like it is easier for newbies to have the examples first. And with the contents menu the experienced user can jump to the api right away